### DISCUSSION TITLE
Clarification about Precursor.Quantity vs MS1.Area vs Fragment.Quant columns

### DISCUSSION THREAD

**Discussion by buijt** [vdemichev/DiaNN#951] [Q&A] [answered]
Hello,

Firstly, thank you for developing DIA-NN and answering questions here on GitHub. I use the DIA-NN software often and I appreciate the work you have put into it.

I am trying to perform my own protein and peptide summarization using the information provided in DIANN_report.tsv (main report) and of course, I would like to calculate the protein quantities based on MS2 quantities, and have some confusion. In the README, I see the following explanations:

> - **Precursor.Quantity** MS2-based quantity of the precursor (charged peptide), non-normalised
> ...
> - **Ms1.Area** non-normalised MS1 peak area
> ...
> - **Fragment.Quant.Raw** raw uncorrected quantities for the top fragment ions, ordered by their reference intensities. Use --report-lib-info to obtain fragment annotation

Typically, "precursor" refers to a detection in the MS1 scan, whereas a "fragment" is an ion detected in the MS2 scan. Therefore, my understanding is that "Fragment.Quant.Raw" and "Fragment.Quant.Corrected" represent the MS2 ions detected in the experiment.

However, I am confused about the difference between the "Precursor" columns and the "Ms1.Area" columns. Precursors are detected in the MS1 scan, so I am confused about the explanation that this is an "MS2-based quantity of the precursor" instead of being described as an MS1 quantity. What is an "MS2-based quantity" and how is it determined?

Additionally, there is a separate column for "Ms1.Area" for the precursor. How is this measurement different than the "Precursor.Quantity" if they are both representative of the identical MS1 detection?

Finally, if I seek to calculate protein quantities myself, which column should I use for quantitative information?

Thank you kindly in advance for your help.

**Comment by vdemichev** [comment, marked as answer]
Hi Jonathan,

In DIA-NN docs (also applies to much of DIA publications), the term 'precursor' also refers to a charged peptide species and is the basic unit of identification and quantification in DIA, hence we also talk about MS2 precursor quantities. Precursors in DIA can be detected and quantified even without any MS1 information, in fact we used to acquire some ultra-fast runs without doing MS1 scans at all.

Please also see the DIA-NN paper (Nat Methods 2020) and QuantUMS preprint (biorxiv 2023) for more details on how DIA-NN quantifies things.

Which column you'd like to use - depends on why you want to do this.
For protein calc based on MS2 quantities, the best would be MaxLFQ with iq package using Precursor.Normalised. This is the best way to process the data if you want to do precursor-level batch correction or apply some special precursor-level filtering. You might then also see if protein-level normalisation is beneficial. But it's strongly recommended to use DIA-NN's normalised precursor quantities or somehow incorporate its normalisation factors (can be calculated as Precursor.Normalised / Precursor.Quantity) in your workflow, as local RT-dependent normalisation is highly helpful in many experiments - corrects for any technical artifacts affecting primarily hydrophilic or in contrast primarily hydrophobic peptides.

Best,
Vadim

**Reply by buijt** [reply]
Thank you Vadim for the detailed answer, and your suggestions. This greatly clarifies my confusion about the terminology and which column I should use for protein quantification.

Best wishes to you,
Jonathan

**Reply by zoe1985** [reply]
> Hi Jonathan,
> 
> In DIA-NN docs (also applies to much of DIA publications), the term 'precursor' also refers to a charged peptide species and is the basic unit of identification and quantification in DIA, hence we also talk about MS2 precursor quantities. Precursors in DIA can be detected and quantified even without any MS1 information, in fact we used to acquire some ultra-fast runs without doing MS1 scans at all.
> 
> Please also see the DIA-NN paper (Nat Methods 2020) and QuantUMS preprint (biorxiv 2023) for more details on how DIA-NN quantifies things.
> 
> Which column you'd like to use - depends on why you want to do this. For protein calc based on MS2 quantities, the best would be MaxLFQ with iq package using Precursor.Normalised. This is the best way to process the data if you want to do precursor-level batch correction or apply some special precursor-level filtering. You might then also see if protein-level normalisation is beneficial. But it's strongly recommended to use DIA-NN's normalised precursor quantities or somehow incorporate its normalisation factors (can be calculated as Precursor.Normalised / Precursor.Quantity) in your workflow, as local RT-dependent normalisation is highly helpful in many experiments - corrects for any technical artifacts affecting primarily hydrophilic or in contrast primarily hydrophobic peptides.
> 
> Best, Vadim

Hi, Vadim,

I'm not clear why you recommended "Precursor.Normalised / Precursor.Quantity"? As you also first suggest that the best would be MaxLFQ with iq package using Precursor.Normalised. 
When processing protein quantification from report.tsv, I normally use Precursor.Normalised, is that right?
When directly use pg.matix, I will just use MaxLFQ?

Best,
Zoe

**Comment by michelc80045** [comment]
Hi Vadim,

Thank you for developing DIA-NN and answering questions. I am new to using DIA-NN but I just wanted to double check that it is functioning properly in relation to the precursor.normalized quantities and the values reported in the PG.Normalized and PG.MaxLFQ in the TSV. File and paraquet files to make sure that the report outputs are functioning properly. 

1) It seems that the reported values for the PG.Normalised column is just the most abundant precursor.Normalised value out of all the precursors found for a specific protein group and specific file in the .TSV report.  I do see in the Paraquet report that it looks like the PG.Normalised is left blank. I assume this is intentional since MaxLFQ is the preferred method?

2) I understand that the PG.MaxLFQ values are the recommended values to use for quantitation and this is a matrix calculation based on the 2014 paper from Cox and Mann using delayed normalization and then peptide ratios among shared peptides across data files and that i will not be able to calculate this myself.  I also understand that we should be filtering the paraquet file based on Lib.Q.Value, Lib.PG.Q.Value, Q.Value and PG.Q.Value columns. I have attached an example below where all the 0.01 filters are satisfied for 7 proteotypic precursors (7/20 total across all data files) that are found in that data file,  and I am surprised that the PG.MaxLFQ value is so close to the most abundant Precursor.Normalized value in the TSV. file. Is this typically observed? This was run with 30 other samples and MBR was used. 

![MaxLFQ table](https://github.com/user-attachments/assets/3d6a630f-6f11-43bf-89b6-49857dfd32fa)

Apologies if this is normal, i just do not have much experience with DIA-NN quite yet. 

Thanks for your thoughts!

Cole

**Comment by vdemichev** [comment]
Hi Cole,

There will be more detailed documentation for DIA-NN 2.0 this week :)

1. It will be blank if you use QuantUMS as QuantUMS precursor quantities are not meant to be used in combination with the top N protein quant method. This will be documented in DIA-NN 2.0 user guide.
2. It's the 'MaxLFQ' basic idea but the actual algorithm is different. No reason why the quantity should not be close the that of a precursor. The whole idea of QuantUMS is that it assigns different weights to different precursors when quantifying proteins, with priority given to the most reliable precursor quantities.

Best,
Vadim
